plugins/removeformat/plugin.js

```diff
@@ -35,6 +35,11 @@ CKEDITOR.plugins.removeformat = {
 					range;
 
 				while ( ( range = iterator.getNextRange() ) ) {
+					// Prevent of removing `span` element with `lang` and `dir` attribute (#779).
+					editor.addRemoveFormatFilter( function( element ) {
```

Comment:
Fix is correct, but it's in incorrect place. It should be inside `language` plugin. There is no need for `removeformat` to know anything about `language` plugin.